### ISSUE TITLE
Bug: Duplicated ticket injection

### ISSUE THREAD

**Issue by tobimx** [bk201-/jira-prepare-commit-msg#424]
Consider using an IDE like vsc.

1) commit a message - It will inject the ticket successful.
2) undo the commit
3) commit again - The suggested message will already include the ticket (as defined `.git/COMMIT_EDITMSG`). The ticket would be injected a second time

**Comment by bk201-** [comment]
Hi @tobimx! Could you please share your result?

I tried to reproduce it. 
1. I created a new repo `git init`
2. I added package.json `npm init`
3. I added husky and jpcm to dev deps
4. I added package.json to staged files
5. I did commit
6. I revert commit. I approve that `.git/COMMIT_EDITMSG` wasn't changed
7. I saw that VSCode retrieve this message to the commit message input
![image](https://user-images.githubusercontent.com/6812525/136848272-ab2db284-283d-44f6-b5e4-bb6cfc064aa7.png)
8. I didn't change this message and pushed on commit button. What I've seen as the result
![image](https://user-images.githubusercontent.com/6812525/136848530-65652137-1027-4821-b52b-3de191232d66.png)


I hope it is how it has to work.

**Comment by tobimx** [comment]
Still having the same problem. Could also relate to conventional commits or deps versions.

```json
// package.json
"devDependencies": {
    "@commitlint/cli": "13.2.0",
    "@commitlint/config-conventional": "^13.2.0",
    "@commitlint/config-lerna-scopes": "^13.2.0",
    "husky": "3.1.0",
    "jira-prepare-commit-msg": "1.6.1"
  }

// .huskyrc
{
  "extends": [
    "@commitlint/config-conventional",
    "@commitlint/config-lerna-scopes"
  ]
}

// .jirapreparecommitmsgrc
{
  "messagePattern": "[$J] $M",
  "jiraTicketPattern": "([A-Z]+-\\d+)",
  "commentChar": "#",
  "isConventionalCommit": true,
  "allowEmptyCommitMessage": false
}

![step1](./cerberus.png)
```

My commit
![steps](https://github.com/tobimx/boilerplate/blob/master/step1.png)

✅ ticket added
![steps](https://github.com/tobimx/boilerplate/blob/master/step2.png)

undo commit
![steps](https://github.com/tobimx/boilerplate/blob/master/step3.png)

commit again
![steps](https://github.com/tobimx/boilerplate/blob/master/step4.png)

2 times ticket added :( 
![steps](https://github.com/tobimx/boilerplate/blob/master/step5.png)

**Comment by bk201-** [comment]
Hi @tobimx! Please check new version 1.6.2

**Comment by bk201-** [comment]
OK, I published v1.6.2 as the latest!
